### ISSUE TITLE
ul.mm-listview overlays div.mm-header

### ISSUE THREAD

**Issue by MaffooClock** [BeSite/jQuery.mmenu#328]
When clicking on an item to reveal a submenu, the `<ul>` lays on top of the header `<div>`.

I managed to overcome this problem by adding the following style to my own stylesheet:

    div.mm-header + ul.mm-listview.mm-first
    {
        margin-top: 20px;
    }

(Incidentally, while trying to make use of the header add-on, I found that after including the JS and CSS files and adding `header: true` in the options did nothing at all.  But, I suppose this is a separate issue.)

**Comment by FrDH** [comment]
Is there an online example? The examples on the website seem to work OK.

**Comment by MaffooClock** [comment]
Yes, the examples all work fine.  I really looked hard at my own CSS thinking I might accidentally be overriding something, but everything seemed clean.  After troubleshooting for a few hours and getting nowhere, I thought I'd mention it in case someone knew anything about it.

Here's a screenshot:
![Issue 328](http://i.imgur.com/OmJlnZC.png)
You can see the "Location History" link is laying on top of the "Employee Locator" header.

**Comment by FrDH** [comment]
Does it make any difference if you use a different browser?

**Comment by MaffooClock** [comment]
Nope.  Identical results in Safari and Chrome.

**Comment by MaffooClock** [comment]
Hmm, I just noticed in my console that I also have the following errors:

    TypeError: i[e] is undefined (jquery.mmenu.dividers.min.js:8:40)
    TypeError: e[t] is undefined (jquery.mmenu.header.min.js:8:38)

Not sure if it's related.

**Comment by MaffooClock** [comment]
I think I see what my problem is.  This is being used in a Ruby on Rails application.

When using the asset pipeline, there is a directive `require_directory` which will compile all JavaScript files together into one.  Since order matters, I included `jquery.mmenu.min` first and relied on `require_directory` to compile the addons after.

However, I had `require_directory` before `require jquery.mmenu.min`, thinking the compiler would still get the order right.  It did not.  After placing `require_directory` after `require jquery.mmenu.min`, the errors went away and the style problem I mentioned disappeared.

Closing this as user error.
